### PR TITLE
New charts for visualization reports.

### PR DESCRIPTION
**Pull request by gabycampagna**
Please test charts.

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit dcfc515d8ae886d1f9ebbdcf9241c3e59bb2d84d

https://deploy-preview-2727--ilios-frontend.netlify.com

**Comment by gabycampagna** [comment]
Trying to test the following page and I am getting redirected to the dashboard UI:

https://deploy-preview-2727--ilios-frontend.netlify.com/data/courses/973/objectives

**Comment by jrjohnson** [comment]
Yes - the netlify build is production-like and this content is still hidden there.  You can only get to it on developer builds.

File: app/styles/app.scss [hunk 1]
```diff
@@ -49,3 +49,4 @@
 @import 'mixins';
 @import 'newcomponents';
 @import 'layout/layout';
+@import 'components/charts';
```
Comment (jrjohnson) [inline app/styles/app.scss]: Please move this into newcomponents.  Everything is slowly moving there and no new stuff should go into `components`

File: app/styles/newcomponents/chart-bar.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.c3-axis-x > .tick {
+  fill: none; // remove axis labels from x-axis
+}
+
+.c3 path,
+.c3 line {
+  fill: none;
+  stroke: #ddd;
```
Comment (jrjohnson) [inline app/styles/newcomponents/chart-bar.scss]: All colors should be declared in variables.scss and referenced only by `$color` variables in other scss files.  There are already a few different grey options there - will one of those work?

Comment (gabycampagna) [inline app/styles/newcomponents/chart-bar.scss]: Found an existing variable that works well.  Adding it to the sass file.

Comment (gabycampagna) [inline app/styles/newcomponents/chart-bar.scss]: Committed and pushed changes.

**Comment by jrjohnson** [comment]
Looks like there is still a scss issue https://travis-ci.org/ilios/frontend/jobs/209613749#L2000

**Comment by gabycampagna** [comment]
Fixed chart-bar.scss.  Passed scss-lint test on my local.

**Pull request merged**